marklogic-data-hub-central/ui-custom/src/components/ResultsList/ResultsList.tsx

```diff
@@ -93,8 +94,16 @@ const ResultsList: React.FC<Props> = (props) => {
 
   const pageLengths = props?.config?.pageLengths ? props?.config?.pageLengths : [10, 20, 80, 100];
 
-  const {returned, start, pageNumber, pageLength} = searchContext;
+  const {returned, sortOrder, pageNumber, pageLength, handleSort} = searchContext;
 
+
+  const handleSortClick = (e) => {
```

Comment:
Currently, there are three possible sort states:
- Unapplied
- Descending
- Ascending

Currently, when the arrows are clicked, the sorting cycles through the states like this:

Unapplied -> Descending -> Ascending -> Descending -> Ascending...

Expected behavior, when the arrows are clicked, the sorting cycles through the states like this:

Unapplied -> Descending -> Ascending -> Unapplied -> Descending -> Ascending -> Unapplied...